### PR TITLE
Support/LIVE-7574 polkadot tranfer api migration

### PR DESCRIPTION
**Pull request by hzheng-ledger**
### 📝 Description
Polkadot tranfer api migration due to the PR https://github.com/paritytech/substrate/pull/12951

### ❓ Context

- **Impacted projects**: `` LLC
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** Only regression tests needed. We check no breaking change for the migration.
- [X] **Atomic delivery** 
- [X] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: libs/coin-polkadot/src/js-buildTransaction.ts
Comment (haammar-ledger) [inline libs/coin-polkadot/src/js-buildTransaction.ts:29]: Camel case probably?

Comment (hzheng-ledger) [reply libs/coin-polkadot/src/js-buildTransaction.ts]: fixed. Thanks

**Review by haammar-ledger — APPROVED** [review]

**Review by chabroA — APPROVED** [review]

**Comment by haammar-ledger** [comment]
:warning: **On hold, do not merge**

Live part is done but related impact identified on backend side and must be addressed too before we merge this (see linked ticket)

**Pull request merged**
